src/applications/disability-benefits/all-claims/pages/addConditionsPages.js

```diff
@@ -0,0 +1,373 @@
+    VA:
+      'Caused by an injury or event that happened when I was receiving VA care.',
+  };
+};
+
+/** @type {ArrayBuilderOptions} */
+const arrayBuilderOptions = {
+  arrayPath: 'newConditions',
+  nounSingular: 'condition',
+  nounPlural: 'conditions',
+  required: true,
+  isItemIncomplete: item =>
+    !item?.condition ||
+    !item?.cause ||
+    (item?.cause === 'NEW' && !item?.primaryDescription) ||
+    (item?.cause === 'SECONDARY' &&
+      (!item?.causedByCondition || !item?.causedByConditionDescription)) ||
+    (item?.cause === 'WORSENED' &&
+      (!item?.worsenedDescription || !item?.worsenedEffects)) ||
+    (item?.cause === 'VA' &&
+      (!item?.vaMistreatmentDescription ||
+        !item?.vaMistreatmentLocation ||
+        !item?.vaMistreatmentDate)),
+  maxItems: 100,
+  text: {
+    getItemName: item => item?.condition,
+    cardDescription: item =>
+      createCauseDescriptions(item?.causedByCondition)[(item?.cause)],
+  },
+};
+
+/** @returns {PageSchema} */
+const introPage = {
+  uiSchema: {
+    ...titleUI(
+      'New conditions',
+      'On the next few screens, we’ll ask you about the new conditions you’re claiming. You must add at least one condition.',
+    ),
+  },
+  schema: {
+    type: 'object',
+    properties: {},
+  },
+};
+
+const isDuplicate = formData => {
+  const { newConditions } = formData;
+
+  if (!newConditions || newConditions.length < 2) {
+    return false;
+  }
+
+  const conditionNames = newConditions.map(condition =>
+    condition.condition?.toLowerCase().trim(),
+  );
+
+  return new Set(conditionNames).size !== conditionNames.length;
+};
+
+/** @returns {PageSchema} */
+const addConditionPage = {
+  uiSchema: {
+    ...arrayBuilderItemFirstPageTitleUI({
+      title: 'Name of new condition you want to claim',
+      nounSingular: arrayBuilderOptions.nounSingular,
+      description: ConditionInstructions,
+    }),
+    condition: {
+      'ui:title': 'Enter your condition',
+      'ui:field': data => (
+        <Autocomplete
+          availableResults={Object.values(disabilityLabelsRevised)}
+          debounceDelay={200}
+          id={data.idSchema.$id}
+          formData={data.formData}
+          onChange={data.onChange}
+        />
+      ),
+      'ui:errorMessages': {
+        required: missingConditionMessage,
+      },
+      // 'ui:validations': [validateDisabilityName], // This is a better solution for the duplicate validation but appStateData is undefined
+    },
+    'view:duplicateAlert': {
+      'ui:description': DuplicateAlert,
+      'ui:options': {
+        hideIf: formData => !isDuplicate(formData),
+      },
+    },
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      condition: {
+        type: 'string',
+      },
+      'view:duplicateAlert': {
+        type: 'object',
+        properties: {},
+      },
+    },
+    required: ['condition'],
+  },
+};
+
+/** @returns {PageSchema} */
+const causePage = {
```

Comment:
Just a comment for us to watch out for, don't expect you to address this. 

I think this is just a design system component thing because I've seen it elsewhere in the form, but the labels for the radio buttons show up twice in Form Controls and the yes/no radio buttons need a better label so you know what you're answering yes/no to.
![IMG_8667](https://github.com/user-attachments/assets/d8eab45f-d26e-468a-84b5-c134bda31753)
![IMG_8669](https://github.com/user-attachments/assets/b1aef3cb-2045-4d44-ad05-3824cae00ce5)